python/src/aac/__init__.py

```diff
@@ -13,7 +13,7 @@
 import os
 
 
-__version__ = "0.3.17"
+__version__ = "0.3.19"
```

Comment:
```suggestion
__version__ = "0.3.20"
```

have two PRs before this one that should be getting to go in.